data/json/items/gun/9mm.json

```diff
@@ -251,6 +252,7 @@
     "longest_side": "377 mm",
     "skill": "pistol",
     "modes": [ [ "DEFAULT", "semi-auto", 1 ] ],
+    "range": 14,
```

Comment:
```suggestion
```
Copies from MP5, Clang is mad you defined duplicate value 😛

Reply:
Thanks!